### ISSUE TITLE
Windows: openslam_gmapping installs dll into lib instead of bin

### ISSUE THREAD

**Issue by robogeekcanada** [RoboStack/ros-noetic#167]
When testing Turtlebot3 SLAM a few errors pop up:

```
$ set TURTLEBOT3_MODEL=burger
$ roslaunch turtlebot3_slam turtlebot3_slam.launch
```
![T3_SLAM error1](https://user-images.githubusercontent.com/32141561/133907414-9e72f013-4bda-4676-86cb-8f5937fc7fb7.PNG)

![T3_SLAM error2](https://user-images.githubusercontent.com/32141561/133907420-d480e9cf-0ff3-45ca-ba62-a574fd8df938.PNG)

![T3_SLAM error3](https://user-images.githubusercontent.com/32141561/133907421-0361bd86-de48-452e-abe2-d3ab5b8b8f37.PNG)

![T3_SLAM error4](https://user-images.githubusercontent.com/32141561/133907423-82076a3c-31db-4221-90df-6951d47a6fbc.PNG)

In summary the following `dlls` not found:

utils.dll, sensor_range.dll, sensor_odometry.dll, scanmatcher.dll

**Comment by robogeekcanada** [comment]
Doing a search in my directory the dlls are installed in 2 places:

```
C:\ProgramData\Miniconda3\envs\robostackenv\Library\lib\openslam_gmapping

C:\ProgramData\Miniconda3\pkgs\ros-noetic-openslam-gmapping-0.2.1-py38h4b9bc1a_5\Library\lib\openslam_gmapping
```

So for some reason ROS is not detecting them

**Comment by robogeekcanada** [comment]
Not sure this is the correct way to fix the problem but it works 😄. I copied the missing dlls into the `bin` folder of my environment.

`C:\ProgramData\Miniconda3\envs\robostackenv\Library\bin`

**Comment by Tobias-Fischer** [comment]
That's a bug in openslam_gmapping. Let's repopen to keep track of it.

**Comment by Tobias-Fischer** [comment]
Fixed thanks to https://github.com/ros-perception/openslam_gmapping/pull/34
